src/python/grapl-tests-common/grapl_tests_common/clients/engagement_edge_client.py

```diff
@@ -15,7 +15,7 @@ class EngagementEdgeException(Exception):
 
 class EngagementEdgeClient:
     def __init__(self, use_docker_links: bool = False) -> None:
-        hostname = "grapl-engagement-edge" if use_docker_links else "localhost"
+        hostname = "auth.grapl.test" if use_docker_links else "localhost"
```

Comment:
technically we should probably consume GRAPL_AUTH_HOST here, but, eh, whatever

Reply:
Fair point.

Reading it again, the `use_docker_links` parameter name doesn't really fit well anymore, either. 🤔